src/pilot/dell_nfv.py

```diff
@@ -0,0 +1,426 @@
+import cpu_siblings
```

Comment:
Put License info at the start of file, pep8 check is failing

Reply:
Put one blank line after the license part

Reply:
Fixed